scripts/ci/style-linter.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
```

Comment:
I don't this this script is really necessary; the `stylelint` cli tool is pretty fast (I just tried it), so we should be able to treat it the same we we currently treat `tslint`, which is to just have an npm script like
```json
"stylelint" : "stylelint 'src/**/*.scss' --syntax scss",
```
(plus specifying the config file)